<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchiveDialog</class>
 <widget class="QDialog" name="ArchiveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>779</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>779</width>
    <height>220</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Архів моделей</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/pics/view_archive.png</normaloff>:/pics/view_archive.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTableView" name="models_table_view">
     <property name="minimumSize">
      <size>
       <width>481</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>481</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="pic_lbl">
       <property name="minimumSize">
        <size>
         <width>267</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>267</width>
         <height>150</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="pics.qrc">:/pics/default_pic.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="restore_pb">
         <property name="text">
          <string>Відновити</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_pb">
         <property name="text">
          <string>Скасувати</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
